sci-mathematics/netgen depends on metis

Got an error while compiling parallelmesh.cpp due to metis

```
/bin/sh ../../libtool  --tag=CXX   --mode=compile x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I. -I../..  -I/usr/local/include -DMETIS -I../../libsrc/include  -I/usr/include/togl-1.7  -march=native -O2 -pipe -mavx -mmmx -msse -msse2 -msse3 -mssse3 -msse4.1 -msse4.2 -mcx16 -mpopcnt -DPARALLEL -c -o paralleltop.lo paralleltop.cpp
parallelmesh.cpp:13:19: fatal error: metis.h: No such file or directory
 #include <metis.h>
                   ^
compilation terminated.
Makefile:571: recipe for target 'parallelmesh.lo' failed
make[3]: *** [parallelmesh.lo] Error 1
make[3]: *** Waiting for unfinished jobs....
libtool: compile:  x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I. -I../.. -I/usr/local/include -DMETIS -I../../libsrc/include -I/usr/include/togl-1.7 -march=native -O2 -pipe -mavx -mmmx -msse -msse2 -msse3 -mssse3 -msse4.1 -msse4.2 -mcx16 -mpopcnt -DPARALLEL -c paralleltop.cpp  -fPIC -DPIC -o .libs/paralleltop.o
make[3]: Leaving directory '/var/tmp/portage/sci-mathematics/netgen-5.0.0/work/netgen-5.0.0/libsrc/meshing'
Makefile:394: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory '/var/tmp/portage/sci-mathematics/netgen-5.0.0/work/netgen-5.0.0/libsrc'
Makefile:440: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/var/tmp/portage/sci-mathematics/netgen-5.0.0/work/netgen-5.0.0'
Makefile:371: recipe for target 'all' failed
make: *** [all] Error 2
 * ERROR: sci-mathematics/netgen-5.0.0::science failed (compile phase):
 *   emake failed
 * 
 * If you need support, post the output of `emerge --info '=sci-mathematics/netgen-5.0.0::science'`,
 * the complete build log and the output of `emerge -pqv '=sci-mathematics/netgen-5.0.0::science'`.
 * The complete build log is located at '/var/tmp/portage/sci-mathematics/netgen-5.0.0/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/sci-mathematics/netgen-5.0.0/temp/environment'.
 * Working directory: '/var/tmp/portage/sci-mathematics/netgen-5.0.0/work/netgen-5.0.0'
 * S: '/var/tmp/portage/sci-mathematics/netgen-5.0.0/work/netgen-5.0.0'

>>> Failed to emerge sci-mathematics/netgen-5.0.0, Log file:

>>>  '/var/tmp/portage/sci-mathematics/netgen-5.0.0/temp/build.log'

 * Messages for package sci-mathematics/netgen-5.0.0:

 * ERROR: sci-mathematics/netgen-5.0.0::science failed (compile phase):
 *   emake failed
 * 
 * If you need support, post the output of `emerge --info '=sci-mathematics/netgen-5.0.0::science'`,
 * the complete build log and the output of `emerge -pqv '=sci-mathematics/netgen-5.0.0::science'`.
 * The complete build log is located at '/var/tmp/portage/sci-mathematics/netgen-5.0.0/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/sci-mathematics/netgen-5.0.0/temp/environment'.
 * Working directory: '/var/tmp/portage/sci-mathematics/netgen-5.0.0/work/netgen-5.0.0'
 * S: '/var/tmp/portage/sci-mathematics/netgen-5.0.0/work/netgen-5.0.0'

```

Metis is installed:
```
$ emerge -s metis
  
[ Results for search key : metis ]
Searching...

*  sci-libs/metis
      Latest version available: 4.0.3
      Latest version installed: [ Not Installed ]
      Size of files: 511 KiB
      Homepage:      http://www-users.cs.umn.edu/~karypis/metis/metis/
      Description:   A package for unstructured serial graph partitioning
      License:       free-noncomm

*  sci-libs/parmetis
      Latest version available: 3.2.0
      Latest version installed: 3.2.0
      Size of files: 5 168 KiB
      Homepage:      http://www-users.cs.umn.edu/~karypis/metis/parmetis/
      Description:   Parallel graph partitioner
      License:       free-noncomm

```

But metis.h is located /usr/include/metis and not /usr/include

--- a/sci-mathematics/netgen/netgen-5.0.0.ebuild
+++ b/sci-mathematics/netgen/netgen-5.0.0.ebuild
@@ -21,7 +21,6 @@ DEPEND="dev-tcltk/tix
 	dev-tcltk/togl:1.7
 	virtual/opengl
 	x11-libs/libXmu
-	
 	opencascade? ( sci-libs/opencascade )
 	ffmpeg? ( media-video/ffmpeg )
 	jpeg? ( virtual/jpeg )
@@ -47,7 +46,6 @@ src_configure() {
 		myconf="${myconf} --enable-occ --with-occ=$CASROOT"
 		append-ldflags -L$CASROOT/lin/$(get_libdir)
 	fi
-	
 	if use mpi; then
 		myconf="${myconf} --enable-parallel"
 		append-cppflags -I/usr/include/metis
@@ -55,7 +53,6 @@ src_configure() {
 	
 	use ffmpeg && myconf="${myconf} --enable-ffmpeg"
 	use jpeg && myconf="${myconf} --enable-jpeglib"
-
 	append-cppflags -I/usr/include/togl-1.7
 
 	econf \

--- a/sci-mathematics/netgen/netgen-5.0.0.ebuild
+++ b/sci-mathematics/netgen/netgen-5.0.0.ebuild
@@ -50,7 +50,6 @@ src_configure() {
 		myconf="${myconf} --enable-parallel"
 		append-cppflags -I/usr/include/metis
 	fi
-	
 	use ffmpeg && myconf="${myconf} --enable-ffmpeg"
 	use jpeg && myconf="${myconf} --enable-jpeglib"
 	append-cppflags -I/usr/include/togl-1.7

--- a/sci-mathematics/netgen/netgen-5.0.0.ebuild
+++ b/sci-mathematics/netgen/netgen-5.0.0.ebuild
@@ -21,7 +21,7 @@ DEPEND="dev-tcltk/tix
 	dev-tcltk/togl:1.7
 	virtual/opengl
 	x11-libs/libXmu
-	|| (sci-libs/parmetis sci-libs/metis)
+	|| ( sci-libs/parmetis sci-libs/metis )
 	opencascade? ( sci-libs/opencascade )
 	ffmpeg? ( media-video/ffmpeg )
 	jpeg? ( virtual/jpeg )

--- a/sci-mathematics/netgen/netgen-5.0.0.ebuild
+++ b/sci-mathematics/netgen/netgen-5.0.0.ebuild
@@ -21,11 +21,11 @@ DEPEND="dev-tcltk/tix
 	dev-tcltk/togl:1.7
 	virtual/opengl
 	x11-libs/libXmu
-	|| ( sci-libs/parmetis sci-libs/metis )
+	
 	opencascade? ( sci-libs/opencascade )
 	ffmpeg? ( media-video/ffmpeg )
 	jpeg? ( virtual/jpeg )
-	mpi? ( virtual/mpi ) "
+	mpi? ( virtual/mpi ( || ( sci-libs/parmetis sci-libs/metis ) ) ) "
 RDEPEND="${DEPEND}"
 # Note, MPI has not be tested.
 
@@ -47,13 +47,16 @@ src_configure() {
 		myconf="${myconf} --enable-occ --with-occ=$CASROOT"
 		append-ldflags -L$CASROOT/lin/$(get_libdir)
 	fi
-
-	use mpi && myconf="${myconf} --enable-parallel"
+	
+	if use mpi; then
+		myconf="${myconf} --enable-parallel"
+		append-cppflags -I/usr/include/metis
+	fi
+	
 	use ffmpeg && myconf="${myconf} --enable-ffmpeg"
 	use jpeg && myconf="${myconf} --enable-jpeglib"
 
 	append-cppflags -I/usr/include/togl-1.7
-	append-cppflags -I/usr/include/metis
 
 	econf \
 		${myconf}

--- a/sci-mathematics/netgen/netgen-5.0.0.ebuild
+++ b/sci-mathematics/netgen/netgen-5.0.0.ebuild
@@ -21,6 +21,7 @@ DEPEND="dev-tcltk/tix
 	dev-tcltk/togl:1.7
 	virtual/opengl
 	x11-libs/libXmu
+	|| (sci-libs/parmetis sci-libs/metis)
 	opencascade? ( sci-libs/opencascade )
 	ffmpeg? ( media-video/ffmpeg )
 	jpeg? ( virtual/jpeg )
@@ -52,6 +53,7 @@ src_configure() {
 	use jpeg && myconf="${myconf} --enable-jpeglib"
 
 	append-cppflags -I/usr/include/togl-1.7
+	append-cppflags -I/usr/include/metis
 
 	econf \
 		${myconf}

